spec/test_async.js

```diff
@@ -0,0 +1,22 @@
+// async await is supported on Node.js 8.10 on AWS lambdas
```

Comment:
NABD but you could `require('./test_callback')` and have the promise and async versions build on that instead of reimplementing the same behaviour.

Reply:
True but I preferred having it this way for greater clarity.